src/Abp.AutoMapper/AutoMapper/AbpAutoMapperModule.cs

```diff
@@ -38,44 +35,25 @@ public override void PostInitialize()
 
         private void CreateMappings()
         {
-            lock (SyncObj)
+            Action<IMapperConfigurationExpression> configurer = configuration =>
             {
-                Action<IMapperConfigurationExpression> configurer = configuration =>
+                FindAndAutoMapTypes(configuration);
+                foreach (var configurator in Configuration.Modules.AbpAutoMapper().Configurators)
                 {
-                    FindAndAutoMapTypes(configuration);
-                    foreach (var configurator in Configuration.Modules.AbpAutoMapper().Configurators)
-                    {
-                        configurator(configuration);
-                    }
-                };
+                    configurator(configuration);
+                }
+            };
 
-                if (Configuration.Modules.AbpAutoMapper().UseStaticMapper)
```

Comment:
@hikalkan this looks like a breaking change for existing user since we no longer respect the value in `UseStaticMapper`

Reply:
@ryancyq you are right, I will mark the issue with breaking change label.